package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Represent minecraft windows",
   "main": "index.js",
   "scripts": {
-    "test": "npm run lint",
+    "test": "npm run lint && mocha",
```

Comment:
Test's in prismarine projects are usually implemented with scripts that look like this 
```json
"mocha_test": "mocha --reporter spec --exit",
"test": "npm run mocha_test",
```
See mineflayers package.json file https://github.com/PrismarineJS/mineflayer/blob/master/package.json

Reply:
implemented.